packages/patterns/package.json

```
@@ -31,6 +31,7 @@
   },
   "main": "dist/index.js",
   "module": "dist-module/index.js",
+  "types": "dist-types/index.d.ts",
   "source": "src/index.js",
```

```suggestion
  "source": "src/index.ts",
```
Same above as well for `src/index.js` reference.

---

Good catch!